src/compiler.ts

```diff
@@ -58,8 +58,13 @@ export class NexeCompiler {
     this.log.step('nexe ' + version, 'info')
     if (python) {
       if (isWindows) {
-        this.env.PATH = '"' + dequote(normalize(python)) + '";' + this.env.PATH
```

Comment:
@calebboyd et al: if you don't like the "shuffling" here, we can make use of the [`caseless` module](https://www.npmjs.com/package/caseless) instead.